components/List/index.jsx

```diff
@@ -0,0 +1,29 @@
+import WorkoutCard from "../Workouts/WorkoutCard";
+import useSWR from "swr";
+
+export default function List({ objectList }) {
+  const fetcher = (url) => fetch(url).then((res) => res.json());
+  const {
+    data: exercises,
+    isLoading,
+    error,
+  } = useSWR(`/api/exercises/`, fetcher);
+  if (isLoading) return <h2>loading</h2>;
+  if (error) return <h2>error</h2>;
+
+  return (
+    <>
+      {objectList.map((workout, index) => (
+        <WorkoutCard
+          key={index}
```

Comment:
```suggestion
          key={workout._id}
```
Use `workout._id` as a key here.
`index` is stable only if the list is completely static. Which in a real world app, is rarely the case.